### ISSUE TITLE
Fieldflow: larger checks and radios

### ISSUE THREAD

**Issue by jmealing** [wet-boew/GCWeb#1548]
We've (DTO-BTN) been working on larger style radios and checkboxes for fieldflow as a result from our optimization project with CRA - which involves some new css and changes to fieldflow - to make it easier and more accessible for people to see and use. 

The next step is creating a PR to make this new design happen, does anyone here see any issues with our css/fieldflow updates we should correct now? We've also reached out to Toufic S at IRCC to review as well.

[https://canada-ca.github.io/design-system-systeme-conception/working-on/selectors-radios-checks/large-checks-radios-fieldflow.html](See the large checkbox and radio designs)

@patlaj @Borges-Porteous

Thanks!

**Comment by lisafast** [comment]
@nschonni @shawnthompson Appreciate your thoughts on this improvement - see the code on the linked page above.

**Comment by nschonni** [comment]
Didn't dig super deep, but here is what I hit
- This are LIs with Links when JS is disabled, instead of native elements (maybe a fieldflow thing, rather than the custom element)
- I use Stylish for custom CSS, and I ran into the issue mentioned on the UK site where with my high contrast CSS, I couldn't tell which radio was selected. Could kind of see it for the checkboxes.
- Tabbing through the checkboxes, jQuery Validation kicked in without a submit. This might have been something WET changed since I stopped following the project
- "Continue" button after checkbox list was tab-able, but the focus indicator was a minor hue change and didn't have a focus indicator. Not visible at all with custom stylesheet

**Comment by jmealing** [comment]
Thanks @nschonni 

- The LI's when JS is disabled is a fieldflow thing, not part of this new css (something to look at later though)

- Stylish - I'll install Stylish and see what can be done for that contrast issue.

- Tabbing through, jQuery Validation - I'll try to reproduce and fix if I can. Could be a WET thing...

- "Continue" focus: I hadn't changed any of the default focus on the current buttons, only for checkboxes/radios. Good catch, I will change my css for the button focus and recommend we fix button focus universally.

**Comment by thekodester** [comment]
jQuery was set to validate when the user tabs out of an input field. So it is a WET thing.

**Comment by thekodester** [comment]
Also for some context there is a discussion about this here: https://github.com/wet-boew/wet-boew/issues/8353

**Comment by nschonni** [comment]
> * Stylish - I'll install Stylish and see what can be done for that contrast issue.

Sorry, I got mixed up with the Gov.UK posts. The post dealing with the contrast issues is https://accessibility.blog.gov.uk/2018/08/01/supporting-users-who-change-colours-on-gov-uk/

**Comment by duboisp** [comment]
@jmealing is it possible for you to come at one of the Tuesday code sprint? So I will be able to review your propose changes.

You PR will need to include the code change for fieldflow, the new CSS, the documentation and working examples in E&F. 

Meanwhile, @jmealing is it possible to comment a sample of HTML markup + CSS (one for radio and one for checkbox) on how web  author will be able to use that new feature? It will ease the peer review process.

**Comment by jmealing** [comment]
@duboisp Yes, we'd love to come to a code sprint. Right now our Tuesday afternoons are booked up for the next few weeks with workshops, but once we are free we will stop by to discuss.

Thanks!

**Comment by GormFrank** [comment]
PR related #1584

**Comment by GormFrank** [comment]
Step 1 of 2 completed: https://github.com/wet-boew/GCWeb/pull/1709
Step 2 of 2 coming soon: @delisma
